src/main/kotlin/com/vk/modulite/settings/ModuliteSettingsState.kt

```diff
@@ -0,0 +1,31 @@
+package com.vk.modulite.settings
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.openapi.components.service
+import com.intellij.util.xmlb.XmlSerializerUtil
+
+/**
```

Comment:
Why this comment?

Reply:
I think its okay, because it was useful for me and I suppose it will be useful for someone else